internal/services/containers/container_group_resource.go

```diff
@@ -123,6 +121,19 @@ func resourceContainerGroup() *pluginsdk.Resource {
 
 			"identity": commonschema.SystemAssignedUserAssignedIdentityOptional(),
 
+			"subnets": {
```

Comment:
Probably we shall rename this to be `subnet_ids`.

Reply:
Yes, I was thinking what name to pick. Sometimes we have `subnets`, sometimes `subnet_ids`. Probably legacy namings.